csharp/Clean Code/ToDo/Program.cs

```diff
@@ -5,33 +5,48 @@ namespace ToDo
 {
     internal class Program
     {
-        public static List<string> TL { get; set; }
+        public static List<string> TaskList { get; set; } = new List<string>();
 
         static void Main(string[] args)
         {
-            TL = new List<string>();
-            int variable = 0;
+            const int addTask = 1;
+            const int removeTask = 2; 
+            const int consultTasks = 3; 
+            const int exit = 4;
+            int menuOption = 0;
```

Comment:
When using constants, you wanna follow naming convention for constants in c# which would be PascalCase, additionally, constants can be placed in a different class file containing only constants for easier reference.

However, for you concrete use case I would recommend an enum which is easier to read and maintain for menu options like this:
```suggestion
private enum MenuOption
{
    AddTask = 1,
    RemoveTask = 2,
    ConsultTasks = 3,
    Exit = 4
}
```